### PR TITLE
NRH-145:  add detail view to donations page

### PR DESCRIPTION
**Pull request by benjaminEwhite**
##  What's this PR do?

This PR adds a donation detail view to the donations dashboard. From the list view, you can click on the "Details..." link for any given row and get taken to a detail view that displays additional information.

## How should this be manually tested?

Log in as an org user. Assuming that there contributions have been seeded for the org, got to `/dashboard/donations/` , then click on the "Details..." link for any given row. You'll be taken to the detail view.

### CODE REVIEW

File: spa/cypress/support/commands.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ Cypress.Commands.add('interceptPaginatedDonations', () => {
   const sortableColumns = ['last_payment_date', 'amount', 'contributor_email', 'modified', 'status', 'flagged_date'];
   const filterableColumns = ['created', 'status', 'amount'];
   const api = new ApiResourceList(donationsData, defaultSortBys, sortableColumns);
-  cy.intercept(`${getEndpoint(CONTRIBUTIONS)}**`, (req) => {
+  cy.intercept({ pathname: getEndpoint(CONTRIBUTIONS) }, (req) => {
```
Comment (JeremyGibson) [inline spa/cypress/support/commands.js:85]: Nice

**Review by JeremyGibson — CHANGES_REQUESTED** [review]
Look into the overflow issue with the `Details` link.

**Comment by benjaminEwhite** [comment]
@JeremyGibson can you provide more context on the overflow issue?  I'm not sure what this means.

**Comment by JeremyGibson** [comment]
@benjaminEwhite I put a screenshot in the ticket.

**Comment by benjaminEwhite** [comment]
Temporarily commenting out the failing test, which we'll run down separately. @JeremyGibson re: your comment about overflow in table, we'll also handle that separately as part of ticket in backlog to make table responsive.

**Pull request merged**